tests/charm-minimal/wheelhouse.txt

```diff
@@ -1,3 +1,3 @@
 # We add a wheel that needs to be built from source to validate that this works
-psycopg2;python_version < '3.8'
-psycopg;python_version >= '3.8'
+psycopg2;python_version <= '3.8'
+psycopg;python_version > '3.8'
```

Comment:
I'm not sure that this is correct; we currently build charms against the current version to get the right thing, and this selects different libraries for 3.8 which is a major change for currently stable charms.  Is there a reason that you need psycopg2 at Python 3.8?

Reply:
On the Github runners testing, the "Functional test with LXDs" was failing - see https://github.com/juju-solutions/layer-basic/pull/211/checks?check_run_id=9000572446 & https://github.com/juju-solutions/layer-basic/pull/211/checks?check_run_id=8994920307